xilem/examples/mason.rs

```diff
@@ -140,8 +140,9 @@ fn run(event_loop: EventLoopBuilder) {
         active: false,
     };
 
-    let app = Xilem::new(data, app_logic);
-    app.run_windowed(event_loop, "First Example".into())
+    Xilem::new(data, app_logic)
+        .background_color(Color::rgb8(0x20, 0x20, 0x20))
```

Comment:
This change isn't that obvious when viewing the example. I'd suggest maybe instead updating the `flex` example to have a white background with black text.

Reply:
I'm ready to commit the change to the `flex` and revert this, but are you sure you want white background there? Black buttons would look a bit out of place there until they can be styled too. I think it's fine if the change isn't very obvious visually, it's more important to demonstrate the code. And IMO dark grey looks better than black for window background anyway (at least it's more common), so such change would still have some value. Wdyt?

Reply:
Here's how the three options look:
- Current:
![Screenshot from 2024-07-22 19-35-39](https://github.com/user-attachments/assets/04433185-2c0c-4e0c-b906-b7a720c5977e)
- `Color::rgb8(0x20, 0x20, 0x20)`
![Screenshot from 2024-07-22 19-36-01](https://github.com/user-attachments/assets/fbf83761-216f-4816-aa8f-c36146a7148a)
- `Color::WHITE`
![Screenshot from 2024-07-22 19-36-38](https://github.com/user-attachments/assets/cfffd577-d387-4512-ab6e-12d4efee48a3)

Reply:
Seems like not only Xilem Button view, but also Masonry Button widget do not expose control over its inner label text size. I guess I'll only commit the docstring for `Xilem::background_color`, and leave the examples as is then?